src/sections/dataset/dataset-citation/DatasetCitation.tsx

```diff
@@ -0,0 +1,94 @@
+import { Col, Row, Tooltip, Icon } from 'dataverse-design-system'
+import styles from './DatasetCitation.module.scss'
+
+import { Citation } from '../../../dataset/domain/models/Dataset'
+
+interface DatasetCitationProps {
+  citation: Citation
+}
+
+function mapAuthors(authors: string[]): string[] {
+  return authors.map((author, index) => {
+    // If this is the last author in the array, don't add a semi-colon
+    if (index === authors.length - 1) {
+      return author
+    }
+    // Otherwise, add a semi-colon after the author
+    return `${author}; `
+  })
+}
```

Comment:
I think we can move this logic to an step previous to the CitationComponent, when we map the js-dataverse response to a UI model

Reply:
I'm going to delete it here and we can always go back in history to move it to the mapping code.